docs/filestructure.rst

```diff
@@ -0,0 +1,228 @@
+.. _filestructure:
+
+==================================
+File structure example & first app
+==================================
+
+We will deal here with a suggestion how you could structure your files and folders for your individual atk4 project.
+Let's assume you are planning to create at least several pages with some models and views. This example can be expanded
+and modified to your needs and shows just one concept of how to setup an atk4 project.
+
+
+File structure example
+======================
+
+This file structure is a recommendation and no must. It is a best practice example.
+Feel free to experiment with it and find the ideal file structure for your project.
+
+* config
+
+    * db.php
+
+* public_html
+
+    * vendor (contains all needed composer modules - don't touch them)
+
+        * atk4
+
+            * ...
+
+        * autoload.php
+
+        * ...
+
+    * projectfolder
+
+        * forms
+
+            * exampleForm1.php
+
+            * exampleForm2.php
+
+        * models
+
+            * exampleClass1.php
```

Comment:
PSR-1+PSR-4 Class must be **StudlyCaps**
PSR-1+PSR-4 Method must be in **camelCase**

Reply:
Good point! Will correct that, thanks...